packages/conf-blas/conf-blas.1/opam

```diff
@@ -6,7 +6,8 @@ bug-reports: "https://github.com/ocaml/opam-repository/issues"
 dev-repo: "https://github.com/ocaml/opam-repository.git"
 license: "GPL-2.0"
 build: [
-  ["sh" "-exc" "cc $CFLAGS test.c -lblas"] {os != "darwin" & os != "win32"}
+  ["sh" "-exc" "cc $CFLAGS test.c -lblas"] {os != "darwin" & os != "freebsd" & os != "win32"}
+  ["sh" "-exc" "${CC:-gcc} $CFLAGS test.c -lblas $LDFLAGS"] {os = "freebsd"}
```

Comment:
That's pure [POSIX shell](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02).

GCC seems to include `/usr/local/{include,lib}` by default; at least under FreeBSD -- unlike system clang (but as explained in #7330, system clang is inappropriate here anyway).